templates/single-tapestry.php

```diff
@@ -114,6 +114,7 @@ function enqueue_vue_app_build()
         <script src="<?php echo plugin_dir_url(__FILE__) ?>libs/jscookie.js" type="application/javascript"></script>
         <script src="<?php echo plugin_dir_url(__FILE__) ?>libs/d3.v5.min.js" type="application/javascript"></script>
         <script src="<?php echo plugin_dir_url(__FILE__) ?>libs/h5p-resizer.min.js" charset="UTF-8"></script>
+        <script src="<?php echo plugin_dir_url(__FILE__) ?>libs/dragselect.min.js"></script>
```

Comment:
I've fixed this one - you had this point to a github repo. It's always better to import the script locally instead or at minimum an external CDN that is reliable.